src/update-feeds-loops/submit-transactions.ts

```diff
@@ -117,15 +117,28 @@ export const submitTransaction = async (
           );
         });
         if (!goMulticall.success) {
-          // It is possible (and sometimes expected) that we try to submit a replacement transaction with insufficient
-          // gas price. Because this is intended flow, we catch the transaction error and log an information message
-          // instead.
-          if ((goMulticall.error as any).code === 'REPLACEMENT_UNDERPRICED') {
-            logger.info(`Failed to submit replacement transaction because it was underpriced.`);
-            return null;
+          // It seems that in practice, this code is widely used. We can do a best-effort attempt to determine the error
+          // reason. Many times, the error is acceptable and results from the way Airseeker is designed.
+          const errorCode = (goMulticall.error as any).code;
+          switch (errorCode) {
```

Comment:
Probably I'd not try to make this error codes more verbose because I think they are self-explanatory. Furthermore, it's hard to cover all error codes, see [here](https://github.com/ethers-io/ethers.js/blob/78829050853093bc5291ae78fc5a904044759aa0/src.ts/utils/errors.ts#L68).